ext/tokenizer/tests/token_get_all_variation17.phpt

```diff
@@ -40,7 +41,7 @@ echo "Done"
 ?>
 --EXPECTF--
 *** Testing token_get_all() : with exception keywords ***
-array(81) {
+array(83) {
```

Comment:
This test had bad syntax before in the code passed to `token_get_all` (it was missing a curly bracket). Fixed that up.